### ISSUE TITLE
Issues with Legacy Ping Passthrough

### ISSUE THREAD

**Issue by Legit4K** [GeyserMC/Geyser#674]
**Describe the bug**

The new ping passthrough system appears to take a hit on server performance including TPS and Memory Usage (which spikes up - possible memory leak?)

Additionally to this, the legacy Ping Passthrough system fails to work at all.

**To Reproduce**

Enable ping passthrough with player slots passthrough. Get 3 - 9 bedrock players on and monitor TPS + RAM Usage. 

**Expected behavior**

The new Ping passthrough system should not have a significant impact on performance over a short period of time.

And the Legacy Ping Passthrough system should work.

**Geyser Config**

```
# --------------------------------
# Geyser Configuration File
bedrock:
  address: 0.0.0.0
  port: 19132
  motd1: Test
  motd2: Test
remote:
  address: 127.0.0.1
  port: 25565
  auth-type: offline
floodgate-key-file: public-key.pem
command-suggestions: true
passthrough-motd: true
passthrough-player-counts: true
legacy-ping-passthrough: false
ping-passthrough-interval: 6
max-players: 999
debug-mode: false
general-thread-pool: 32
allow-third-party-capes: true
allow-third-party-ears: false
default-locale: en_us
cache-chunks: true
above-bedrock-nether-building: true
metrics:
  enabled: true
  uuid: <HIDDEN>
config-version: 3
```

**Server Version**

`git-Paper-321`

**Geyser Version**

Type: `Geyser-Bukkit`
Branch: `master`
Build: `202`

**Minecraft: Bedrock Edition Version**

1.14.6(0)

**Additional Context**

This was tested with 6-8 Bedrock Players connected through the Geyser Plugin and 14GB RAM was allocated to the server.

Even still, this probably shouldn't be marked as Confirmed because I may be entirely wrong about the memory leak and performance degrade. However legacy ping passthrough doesn't work for a fact (at least on my machine).

When I disabled the new Ping Passthrough System and switched to using legacy ping passthrough, it didn't forward the MOTD of the server from the ServerListMOTD Plugin, but instead it fetched the MOTD listed in the Geyser Config. However with the legacy ping passthrough, I noticed a performance gain. I've only testing this once so I'm gonna retest 2 or 3 more times for accurate results.

**Comment by towuk118** [comment]
hm... 🤔

**Comment by Legit4K** [comment]
After some more testing, the part about a possible Memory leak is untrue as my stats were giving the wrong readings of ram usage, however I did notice that all geyser players would disconnect after a while.

However, the part about the legacy ping passthrough system is true - it doesn't seem to work at all and instead it fetches the MOTD from the Geyser config file.

**Comment by RednedEpic** [comment]
Resolved in 8ac5d6e13d92e3c8919a410c34d94850491a0861.
